src/components/SubscribeSection/styles.module.css

```diff
@@ -2,20 +2,22 @@
   box-sizing: border-box;
   position: relative;
   display: flex;
+  flex-flow: column nowrap;
   align-items: center;
-  min-height: 300px;
-  padding: 20px 15px 30px;
+  justify-content: center;
   background-color: #13adc7;
-
-  @media (--xs-scr) {
-    min-height: auto;
-  }
 }
 
 .container {
   margin: 0 auto;
+  padding: 85px 15px 65px;
   width: 100%;
   max-width: 1000px;
+  box-sizing: border-box;
+
+  @media (--xs-scr) {
+    padding: 20px 15px;
+  }
```

Comment:
A bit of refactoring to make the whole component more receptive to additional vertical content